src/libraries/BLS.sol

```diff
@@ -104,7 +106,7 @@ library BLS {
         bool success;
 
         assembly {
-            success := staticcall(sub(gas(), 2000), 6, input, 0xc0, r, 0x60)
```

Comment:
I don't think there's a scenario where it's useful to reserve gas like that so I left it off